app/utilities/text.py

```diff
@@ -0,0 +1,19 @@
+def trim_with_ellipsis(text, max_length):
+    """
+    checking if provided string is longer than given value, if so then it needs to be cut and 3 dots need to be added
+    """
+    if len(text) > 12:
```

Comment:
Don't forget to use the `max_length` parameter
```python
if len(text) > max_length and max_length > 3:
    text = text[0:max_length - 3] + "..."
```

Reply:
+1 to this.
Does `max_length - 3` need to be in parenthesis when it's used in `[0:max_length - 3]`?